css/zocial.css

```diff
@@ -119,7 +119,7 @@ a.zocial {
 	background-image: -o-linear-gradient(bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
 	background-image: -webkit-gradient(linear, left top, left bottom, from(rgba(255,255,255,.1)), color-stop(30%, rgba(255,255,255,0)), color-stop(50%, transparent), to(rgba(0,0,0,.1)));
 	background-image: -webkit-linear-gradient(bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
-	background-image: linear-gradient(to bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
+	background-image: linear-gradient(bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
```

Comment:
I followed the exact steps described

1 Install Font Custom
2 Add new font in the src/ folder.
3 Set color settings in the templates/zocial.css file.
4 Run fontcustom compile
5 Update the sample.html file with both the button and icon.
6 Test rendering. If broken go to step 2.
7 Send pull-request !

I only added the twitch line to the templates/zocial.css file, added the .svg to the src folder and ran "fontcustom compile"... So the compilation must have done all that.